projects/core/src/i18n/translatable.ts

```diff
@@ -5,11 +5,15 @@
  */
 
 export interface Translatable {
-  key?: string;
+  key?: string | string[];
   params?: TranslatableParams;
   raw?: string;
 }
 
 export interface TranslatableParams {
   [param: string]: any;
 }
+
+export function isTranslatable(input: any): input is Translatable {
```

Comment:
We should probably cover this with a unit test.

Reply:
Add. Thanks for unit tests found one problem in this method :) 
```
typeof input === 'object' &&
    ('key' in input || 'raw' in input)
```

typeof null is also `object` so the part `'key' in input` could make problems.